README.md

```diff
@@ -9,13 +9,15 @@ The app is designed for everyone, from beginners to experienced entrepreneurs an
 
 ## Features
 - **Practice your speech skills**: The user has 3 different practice modes: Sales pitch, public speaking, and interview. The last one works in real-time, meaning the interview questions can change depending on the user's answers thanks to the Chat-GPT API we use.
-- **Real-Time Analysis**: Provides real-time feedback on speech delivery, including aspects such as the emotions that can be derived from the words given as a response (not the tone) as well as what points the user can work on better and how good the response to an interview question/the speech given was.
+- **Real-Time Analysis**: Provides real-time feedback on speech delivery, including aspects such as the emotions that can be derived from the words given as a response as well as what points the user can work on better and how good the response to an interview question/the speech given was.
 - **Friends**: Users can have friends and see how they performed on the same interviews, giving them a sense of competition and a drive to work more on their public speaking skills.
 To add friends, the user should send a request to the person he wants to add and they should accept it, but for now we have a "follow system" where the user can add people as friends and will see them as a friend, but the person added doesn't have them as a friend until they follow back.
```

Comment:
This is not actual anymore, the functionality of having "friends" rather than "followers" was introduced in #186.
```suggestion
To add friends, the user should send a request to the person he wants to add and they should accept it.
```